babel/src/test/resources/sql/spark.iq

```diff
@@ -448,4 +448,177 @@ EXPR$0
 [2.111111111111112, 2.1]
 !ok
 
+# Tests for CALCITE-6680
```

Comment:
Is there any verification in Spark?

Reply:
No, this is based on the spec of these functions in Calcite. I don't have access to a Spark system.
We should build a proper way to test queries on other systems, not do it manually, this is not sustainable.

Reply:
Sure